### DISCUSSION TITLE
Is `native` prop gone in v9?

### DISCUSSION THREAD

**Discussion by pauldijou** [pmndrs/react-spring#1409] [Support] [answered]
In v8, when using the render-props API, you could specify a `native` property on those components which could boost performance according to [the documentation](https://www.react-spring.io/docs/props/performance).

It seems that this property is gone in v9 (couldn't find it in the codebase nor in the TypeScript types) but it was not mentioned in the [breaking changes](https://aleclarson.github.io/react-spring/v9/breaking-changes/) so I just wanted for confirm?

**Comment by joshuaellis** [comment, marked as answer]
I think so yes, the whole render props API has changed so you need to wrap your components in `Animated`. We're in the progress of updating the documentation.
